internal/iam/repository/view/model/idp_config.go

```diff
@@ -63,13 +67,20 @@ func IDPConfigViewToModel(idp *IDPConfigView) *model.IDPConfigView {
 		IsOIDC:                     idp.IsOIDC,
```

Comment:
basically yes, but IMHO changing this now does not make sense
I'd prefer doing this in the query side change, because it would now take too much effort.